app/app.go

```diff
@@ -300,7 +296,8 @@ func (d *Deployments) ProvisionTenant(ctx context.Context, tenant_id string) err
 func (d *Deployments) CreateImage(ctx context.Context,
 	multipartUploadMsg *model.MultipartUploadMsg) (string, error) {
 
-	if multipartUploadMsg.ArtifactSize > MaxImageSize {
+	maxImageSize := config.Config.GetInt64(dconfig.SettingAwsS3MaxImageSize)
```

Comment:
Sorry, I should've called in my first review, but can we do the same here with `Deployments`?

Reply:
No problem, I force-pushed the change.